docs/run-new/index.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Introduction
+description: Overview of the Run section, where you'll find information on Qiskit Runtime primitives, execution modes, IBM Quantum systems, and executing jobs
+
+---
+
+# Introduction
+
+IBM Quantum&trade; maintains the world’s most advanced fleet of quantum systems, with seven [utility-scale](https://www.ibm.com/blog/announcement/new-ibm-quantum-systems-on-the-ibm-cloud/) quantum systems, and more on the way. These systems demonstrate unparalleled reliability, with >95% uptime across the fleet of quantum systems - and unmatched stability, with two-qubit gate error fluctuations no larger than 0.001 over timescales measured in months[^1].
+
+## Run on hardware
+
+Step 3 in Qiskit Patterns, "Run on hardware," centers around the [Qiskit Runtime primitives](/run/primitives) as the interface with IBM Quantum&trade; hardware, abstracting away error suppression and mitigation techniques so that you can focus more on the outcomes of your quantum computation.
```

Comment:
```suggestion
Step 3 in Qiskit Patterns, "Execute on target hardware," centers around the [Qiskit Runtime primitives](/run/primitives) as the interface with IBM Quantum&trade; hardware, abstracting away error suppression and mitigation techniques so that you can focus more on the outcomes of your quantum computation.
```

Reply:
I think this is the naming we're going to have to go with